README.md

```diff
@@ -24,33 +24,29 @@
 
```

Comment:
- [ ] **Required:** Kindly update the README with the appropriate title and description. This would make your project more professional.

Reply:
done :+1: